lib/html_scrubber_util.rb

```diff
@@ -9,7 +9,7 @@ module HtmlScrubberUtil
   def sanitize_html(value)
     ActionController::Base.helpers.sanitize(
       value,
-      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS,
+      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete("select").merge(['style', 'title']),
```

Comment:
Checked with Trey on this line-- we should probably remove `title` from the accepted elements since we don't have an example of any HTML that's actually broken by excluding it.  It was just the PDF generation that was broken.

Reply:
https://github.com/ideacrew/enroll/pull/3596/commits/d3df566ab6238021902a7db5f2a5cf2d67fb4d1e should fix this